src/Init/WFTactics.lean

```diff
@@ -22,7 +22,7 @@ macro_rules | `(tactic| decreasing_trivial) => `(tactic| linarith)
 -/
 syntax "decreasing_trivial" : tactic
 
-macro_rules | `(tactic| decreasing_trivial) => `(tactic| simp (config := { arith := true }); done)
+macro_rules | `(tactic| decreasing_trivial) => `(tactic| (simp (config := { arith := true })); done)
```

Comment:
Why do we need the extra parentheses? Is this some kind of a workaround?

Reply:
It should be unnecessary. I believe this line was modified as part of a stage0 update (adding the flag to the config caused compiled `simp` not to match the new VM layout), and it was incompletely reverted.

Reply:
```suggestion
macro_rules | `(tactic| decreasing_trivial) => `(tactic| simp (config := { arith := true }); done)
```